TF_Service_and_Tooling_Unity/Assets/TouchFree/ServiceShared/Scripts/Configuration/Managers/InteractionConfig.cs

```diff
@@ -38,11 +38,17 @@ public override void SetAllValuesToDefault()
 
             UseScrollingOrDragging = defaults.UseScrollingOrDragging;
             DeadzoneRadius = defaults.DeadzoneRadius;
+
+            interactionZoneEnabled = defaults.interactionZoneEnabled;
```

Comment:
Is it intentional that the interaction zone properties are lowercased, while others are upper case?

Reply:
This has been addressed in another PR, it was accidental and actually caused a bug when requesting config changes from Tooling

Reply:
Ah yes, I see the other PR now. Not a problem for this one in that case.